<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CParamsDialog</class>
 <widget class="QDialog" name="CParamsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sensivity</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="edtSensivity">
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Averaging window length</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="edtWindowLen"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Count of seconds where to take instant. spectrum</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="edtSecondsCount"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CParamsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CParamsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtSensivity</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CParamsDialog</receiver>
   <slot>Changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtWindowLen</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CParamsDialog</receiver>
   <slot>Changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtSecondsCount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CParamsDialog</receiver>
   <slot>Changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Changed()</slot>
 </slots>
</ui>
